src/test/java/bit/javaoop/BudgetManagerTest.java

```diff
@@ -0,0 +1,35 @@
+package bit.javaoop;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class BudgetManagerTest {
+
+    @Test
+    void isHire() {
+    }
+
+    @Test
+    void addEmployee() {
+        List testLista = new ArrayList();
+        Employee testEmployee=new Employee("New Worker",10000);
+        double salary=testEmployee.getSalary();
+        String name=testEmployee.getName();
+        //testEmployee.isHire();
+        //nie wiem co dalej
```

Comment:
No to tak. Na początku formalności:
+ nazwa metodki powinna zaczynać się od `should` i przyjmować konwencję `shouldDoSomething`. Czyli `shouldAddEmployee`
+ komentarze //given //when //then - powinny się również pojawić

A teraz do rzeczy, jak to miało wyglądać. Testujesz metodę addEmployee z klasy `Manager` (swoją drogą skoro ta metoda jest z `Manager`a, to klasa testująca ją powinna się nazywać `ManagerTest`, ale mniejsza :P). A więc gdzieś ją trzeba odpalić! A my tutaj nigdzie jej nie używamy :).

Może od razu napisze jak to miałoby wyglądać, rozwieje to od razu wątpliwości, zamiast pisać latami teorie:
![image](https://user-images.githubusercontent.com/26125971/38339922-0f9e972c-3871-11e8-8d20-3301f34b7c95.png)
No i tłumaczymy:
Najpierw deklaruje sobie testowego managera i employee (blok given)
Potem odpalam metodę testowaną (blok when) - spodziewam się, że doda pracownika do listy pracowników mojego managera.
Na koniec robie assercję (blok then). Sprawdzam w niej czy lista pracowników managera zawiera (contains) pracownika. 
I to tyle :) Po zastosowaniu moich poprawek z komentarzy powyżej test mi przeszedł :P